drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -160,16 +173,16 @@ def items_without_answers():
             "url_name": getattr(self, 'url_name', ''),
             "display_zone_labels": self.data.get('displayLabels', False),
             "display_zone_borders": self.data.get('displayBorders', False),
-            "items": items_without_answers(),
-            "title": self.display_name,
+            "items": items_without_answers(self),
+            "title": self._(self.display_name),
```

Comment:
It is hooked up to pylint ([e.g.](https://travis-ci.org/edx-solutions/xblock-drag-and-drop-v2/builds/109975828)) but it doesn't seem to have caught those. pylint version / pylintrc problem perhaps?

BTW some of the tests are flaky but we have two open PRs that fix the flaky tests.